### DISCUSSION TITLE
Help, timed message sends are not working in AsyncTelebot!

### DISCUSSION THREAD

**Discussion by izzy8799** [eternnoir/pyTelegramBotAPI#2019] [Q&A]
Please answer these questions before submitting your issue. Thanks!

1. What version of pyTelegramBotAPI are you using?
4.12.0

2. What OS are you using?
Void Linux x86_64 (GLibc)

3. What version of python are you using?
3.11.4

Good afternoon. I need to make sending messages at the right time in AsyncTelebot. I tried using aioscheduler and scheduler from instructions on the internet, even through BingAI, but nothing worked. Aioscheduler would start, but would not output anything. Scheduler started, but just at the right time gave an error with await and nothing else.

**Comment by izzy8799** [comment]
So what should I do?

**Reply by Hamiiim** [reply]
```
def scheduled():
    logging.info("==================TeskStart==================")
    while True:
        schedule.run_pending()
        time.sleep(15)


schedule.every(59).minutes.do(scheduled_message)
schedule.every(120).seconds.do(scheduled_delete_message)
polling_thread1 = threading.Thread(target=scheduled)
polling_thread1.start()

bot.polling(allowed_updates=update_types)
```

**Reply by Hamiiim** [reply]
I used additional threads for polling to solve this
